packages/components/src/core/Panel/style.ts

```diff
@@ -0,0 +1,99 @@
+import styled from "@emotion/styled";
+import { Drawer, drawerClasses } from "@mui/material";
+import {
+  CommonThemeProps,
+  getSemanticColors,
+  getShadows,
+  getSpaces,
+} from "../styles";
+import { PANEL_BASIC_MIN_WIDTH, PANEL_OVERLAY_MIN_WIDTH, PanelProps } from ".";
+import { css, SerializedStyles } from "@emotion/react";
+
+type PanelExtraProps = PanelProps & CommonThemeProps;
+
+const doNotForwardProps = [
+  "sdsType",
+  "position",
+  "width",
+  "headerComponent",
+  "onClick",
+  "disableScrollLock",
+  "closeButtonOnClick",
+  "CloseButtonComponent",
+];
+
+const BasicPanel = (props: PanelExtraProps): SerializedStyles => {
```

Comment:
discussion: I'm thinking about naming it `basicPanelStyle`, since it's styles not a component. What do you think?
```suggestion
const basicPanelStyle = (props: PanelExtraProps): SerializedStyles => {
```

Reply:
Hmm, I like that. thanks for the suggestion 😍